src/components/Login.vue

```diff
@@ -5,41 +5,129 @@
+                  </div>
+            </div>
+        </div>
 
-          <!-- <label for="email1">Email</label>
-          <input v-model.trim="loginForm.email" type="text" placeholder="you@email.com" id="email1" />
+        <!--PLAN AHEAD-->
+        <div class="container p-0">
+            <div class="row top-bar no-gutters">
+                <div class="col">
+                    <div class="row no-gutters"><h1 class="plan-head mt-5">Plan Your Courses Ahead</h1></div>
+                    <div class="row no-gutters"><p class="plan-subhead">Introducing the new and easiest way to plan courses at Cornell</p></div>
+                    <div class="row">
+                        <div class="col-5" >
+                            <button  @click="socialLogin"  class="email-button" variant= "primary"  v-on:click="addUser"> GET STARTED </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <img id= "hide"  style="position:relative" class="laptop" src="../assets/images/laptop.png" alt = "laptop preview" />
+                </div>
+            </div>
+        </div>
 
-          <label for="password1">Password</label>
-          <input v-model.trim="loginForm.password" type="password" placeholder="******" id="password1" />
+        <!--NEW WAY-->
+        <div class="new container p-0">
+            <h1 class= "new-1 head" style = "text-align: left;">New Way to Track Your Requirements </h1>
+            <div class="row new no-gutters">
+                <div class="col">
+                    <div class="row tasks">
+                        <div class="col-1 tasks"><img src="../assets/images/Task1.svg" alt = "checklist"/></div>
+                        <div class="col-11"><p class= "sub">Fully personalized to track your requirements</p></div>
+                    </div>
+                    <div class="row tasks">
+                        <div class="col-1 tasks"><img  src="../assets/images/Task2.svg" alt = "browser" /></div>
+                        <div class="col-11"><p class= "sub">Customizable interface to view your courses</p>  </div>
+                    </div>
+                    <div class="row tasks">
+                        <div class="col-1 tasks"><img  src="../assets/images/Task3.svg" alt = "Network" /></div>
+                        <div class="col-11"><p class= "sub">Built-in system to check your progress</p></div>
+                    </div>
+                    <div class="row tasks">
+                        <div class="col-1 tasks"><img  src="../assets/images/Task4.svg" alt = "Starred comment" /></div>
+                        <div class="col-11"><p class= "sub">Recommends courses based on your needs</p> </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <img id= "hide" style="position:absolute" class="women" src="../assets/images/Person_planning.svg" alt = "women planning" />
```

Comment:
Any reason the styling here and in a lot of this file is in the HTML instead of the style tag? Easier to find where the styling is coming from if its within the class.

Reply:
Need this for overlapping photos over sections.